src/components/SwapForm/Form.tsx

```diff
@@ -187,8 +187,8 @@ export default function SwapForm() {
                 user: address,
                 nonce: gaslessNonce,
                 originChainId: chainId,
-                openDeadline: BigInt(1828585485),
-                fillDeadline: BigInt(1828585485),
+                openDeadline: BigInt(Math.floor(Date.now() / 1000) + 600), // 10 minutes from now
+                fillDeadline: BigInt(Math.floor(Date.now() / 1000) + 3600), // 1 hour from now
```

Comment:
no

fillDeadline is slightly misleading since it's more like "reclaim deadline" so if the zk proof doesn't appear for long (post-mocks) the filler can lose money

set 6 hours or something here

Reply:
32e1d44